draft-ietf-tls-tls13.md

```diff
@@ -3164,8 +3184,11 @@ ticket_age_add
 ticket
 : The value of the ticket to be used as the PSK identity.
 The ticket itself is an opaque label. It MAY either be a database
-lookup key or a self-encrypted and self-authenticated value. Section
-4 of {{RFC5077}} describes a recommended ticket construction mechanism.
+lookup key or a self-encrypted and self-authenticated value. It
+MUST NOT be unprotected plaintext that can be modified by the
+recipient or an attacker without being rejected when presented back
+to the server.  Section 4 of {{RFC5077}} describes a recommended
+ticket construction mechanism.
```

Comment:
I'm not sure this is correct. Consider the case where it's a database key and we just issue them in sequence. Is that unsafe?